storage/block/types.go

```diff
@@ -135,6 +135,12 @@ type DatabaseSeriesBlocks interface {
 	// Len returns the number of blocks contained in the collection.
 	Len() int
 
+	// IsSealed returns whether all the blocks are sealed
+	IsSealed() bool
+
+	// MarkSealed marks the series as sealed
+	MarkSealed()
```

Comment:
I'd prefer if we had a method that actually sealed the unsealed rather than someone on the outside sealing the unsealed then telling the blocks collection that they are now sealed.  Seems like better encapsulation.

Perhaps a method `SealAllUnsealed()`?